sub-projects/hub/src/server/aws/aws_s3.js

```diff
@@ -0,0 +1,42 @@
+const AWS = require('aws-sdk');
+const async = require('async');
+const bucketName = 'knownfaces';
+const path = require('path');
+const fs = require('fs');
+let pathParams, image, imageName;
+
+const s3 = new AWS.S3({region: 'us-west-1'});
+const createItemObject = (callback) => {
+  const params = {
+    Bucket: bucketName,
+    Key: '${imageName}',
```

Comment:
Are you trying to use string templates or is the key supposed to look like this? If the former, I have two concerns: I don't think `imageName` is defined and you could get the same result with just `Key: imageName`